logical/plugin/backend.go

```diff
@@ -22,3 +29,23 @@ func (b *BackendPlugin) Server(broker *plugin.MuxBroker) (interface{}, error) {
 func (b BackendPlugin) Client(broker *plugin.MuxBroker, c *rpc.Client) (interface{}, error) {
 	return &backendPluginClient{client: c, broker: broker, metadataMode: b.metadataMode}, nil
 }
+
+func (b BackendPlugin) GRPCServer(broker *plugin.GRPCBroker, s *grpc.Server) error {
+	pb.RegisterBackendServer(s, &backendGRPCPluginServer{
+		broker:  broker,
+		factory: b.Factory,
+		// We pass the logger down into the backend so go-plugin will forward
+		// logs for us.
+		logger: logbridge.NewLogger(b.Logger).LogxiLogger(),
```

Comment:
Where do we set `b.Logger`? I couldn't find any reference on where it's being set.

Reply:
https://github.com/hashicorp/vault/pull/3808/files/bc31a4623c998e6baa66544534ccb14a90bd8ae9#diff-046797d040beeb3fd5d9ff2afedcefc6R42